src/components/SamplePanel/infoRow.js

```diff
@@ -49,8 +49,13 @@ const InfoRow = ({sampleName, sampleData, sampleColour, menuItems, handleClick,
       const readLastSeen = sampleData.readsLastSeen+timeSinceLastDataUpdate;
       if (readLastSeen > 5) {
         const timeFormatter = makeTimeFormatter();
-        summaryText += `| read last seen ${timeFormatter(readLastSeen)} ago`
+        summaryText += `| read last seen ${timeFormatter(readLastSeen)} ago`;
       }
+      let variant_name = sampleVariant.map(variant => variant.name).join(" ");
+      if(variant_name===""){
```

Comment:
this could be tested before and assignedi n a single expression, e.g.
`const variant_name = sampleVariant.length > 0 ? sampleVariant.map().join() : "not known yet"`